src/__experimental__/components/checkable-input/checkable-input.style.js

```diff
@@ -19,13 +20,17 @@ const StyledCheckableInputWrapper = styled.div`
   }) => css`
     ${FormFieldStyle} {
       display: flex;
-      flex-wrap: wrap;
+      flex-flow: row wrap;
```

Comment:
Does the FormFieldStyle have `flex-direction: column`? If not I think that we can just use `flex-wrap: wrap` because the row is a default value for `display flex`. If not, just omit the comment

Reply:
resolved in f37ec57f3d23c58afbcb46ab4bd5cf5672be04c0